### ISSUE TITLE
Error in main lua adv_weapons a nil value

### ISSUE THREAD

**Issue by Mathiaszero** [appgurueu/adv_weapons#2]
ModError: Failed to load and run script from D:\Portal\New Program Files\Games\minetest-5.3.0_dedicatedTest\bin\..\mods\adv_weapons\init.lua:
...est-5.3.0_dedicatedTest\bin\..\mods\adv_weapons/main.lua:1: attempt to index global 'adv_weapons' (a nil value)
stack traceback:
	...est-5.3.0_dedicatedTest\bin\..\mods\adv_weapons/main.lua:1: in main chunk
	...\minetest-5.3.0_dedicatedTest\bin\..\mods\modlib/mod.lua:11: in function 'include_env'
	...\minetest-5.3.0_dedicatedTest\bin\..\mods\modlib/mod.lua:29: in function 'extend'
	...est-5.3.0_dedicatedTest\bin\..\mods\adv_weapons\init.lua:2: in main chunk
Check debug.txt for details.

I am using 5.3.0, it crashed right after I hit connect. I have all dependencies installed. Thanks in advance.

**Comment by appgurueu** [comment]
Cannot reproduce. Try upgrading modlib.

**Comment by Mathiaszero** [comment]
![image](https://user-images.githubusercontent.com/55817193/103604064-12cc0980-4ede-11eb-9f77-e3cf8fcee75b.png)
![image](https://user-images.githubusercontent.com/55817193/103604124-31ca9b80-4ede-11eb-8394-9bd2d104e360.png)

So I removed and re-added/upgraded the two mods by downloading from Content DB the latest versions, and I get the same error. If you can give me about a week I can work to see if other versions have the same issue or try a different minetest build. Otherwise unfortunately it really is just me and I will have to move on.

In the meantime I do want to note that without advanced weapons active, modlib does not produce any errors and my game is running normally. Conversely, if advanced weapons is active and modlib is inactive, my game is also running normally. Somehow the two mods do not seem to like each other.

**Comment by Mathiaszero** [comment]
After tinkering around, I ended up downgrading to rolling 14 of modlib; paired with your latest version of adv_weapons, the mods finally worked together. I had happened to skim through the forum and a user was talking about the mod in its working state which was about that time. I guess case is closed now, I appreciate your patience. Another reason it is nice Github allows you to look for previous versions with ease. Not the best fix, but at least no more crashes.
